### ISSUE TITLE
Future of Sentinelsat after the termination of the Copernicus Open Access Hub

### ISSUE THREAD

**Issue by j08lue** [sentinelsat/sentinelsat#607]
Copernicus Open Access Hub / SciHub, the main endpoint this library was used to query, has been permanently terminated.

It is [succeeded by the Copernicus Data Space Ecosystem (CDSE)](https://github.com/sentinelsat/sentinelsat/issues/583).

I think it will take a while for the community to realize that Sentinelsat is currently not a project to turn to for querying and downloading Copernicus data from official archives. 

Question to past contributors and maintainers of this project (e.g. @valgur, @kr-stn, @avalentino) and aspiring new ones (@fpl, ++) - what should we do about this repo and the PyPi package?

- [ ] 👍 Archive the repo and with it all the issues, docs, etc that refer to the now obsolete SciHub
- [ ] 🚀 Remove the package from PyPi
- [ ] ❤️ Keep the package but release a version where we removed references of SciHub (default URL etc) - maybe there are still DHuS hubs that use the same protocol?
- [ ] 👀 Keep the repo up, but add disclaimers everywhere that Sentinelsat right now does not work and help is appreciated to turn it into a CDSE client?

Please vote with emojis below 👇, in particular any new committed contributors. (Personal note: after closing this issue (and maybe supporting any actions) within the next month or so, I'll be off this project. 👋 )

**Comment by WALEE1789** [comment]
Is there any update on this?

**Comment by fpl** [comment]
I would also add a few rows of warning about the status of the code into the README file on master, even in case of archiving.

**Comment by valgur** [comment]
There are still multiple national mirrors using the legacy DHuS system compatible with `sentinelsat`, so the package should definitely stay on PyPI: https://github.com/kr-stn/awesome-sentinel#data-access

**Comment by yhyxzx** [comment]
How to download data for the new CDSE?

**Comment by fpl** [comment]
> Ohhh! Big problem to solve here. We have forest monitoring and depends on this service to continue.

Well, in #583 there are a few suggestions about some programs that could possibly work now with CDSE. I also managed to write some Perl modules and scripts to replace my now archived https://github.com/fpl/scihub project (if interested, just ask off issue). 
The most complicated aspect of the new APIs is that they are still changing (I had to fix OSearch methods even today), so I suspect it shall require some time to have projects decently and stably working.

**Comment by vanhog** [comment]
I uses Sentinelsat for a while and it did it's job perfectly.  What about sentinelsat_for_CDSE?

**Comment by janceviciusjulius** [comment]
For CDSE You need username and password, where do you create account, can provide link? 

I have created account, but it asked for email and password..

**Comment by subnet001** [comment]
Hi everyone, I'm currently working on a replacement for sentinelsal specifically for CDSE.

https://github.com/SDFIdk/CDSETool

There's a lot missing at the moment but the core functionality is already in place: query and downloading from CDSE. Please check it out, any and all feedback is greatly appreciated!

**Comment by j08lue** [comment]
Decision is made - the repo and package stay up, but we should

- https://github.com/sentinelsat/sentinelsat/issues/611

**Comment by massimozanetti** [comment]
I am sorry in advance if I am asking this question the 100th time... but still I don't get exactly from the open issues and the various discussions therein if `sentinelsat` is going to be updated in the future to support the download of Sentinel products from CDSE. In my center we have a service based on sentinelsat (as many others here and there could have..), should we expect this update, or `sentinelsat` it is officially dismissed for query and download of Sentinel products?

**Comment by kr-stn** [comment]
@massimozanetti you can regard `sentinelsat` to be in maintenance mode _as is_ for connecting to DHuS instances.

There is currently no concrete plan to refactor it to support the CDSE infrastructure. This is mostly due to a lack of time from the existing participants. We welcome anyone who would like to contribute that change.

**Comment by Juan-Pablo-17** [comment]
Hello, my name is Juan. I apologize in advance. I am a Physics student, and I took a course with EO College called "Echoes in Space," which led me to this forum. I learned some basic tools and how to acquire images from the Sentinels. However, I noticed some time ago that the page for acquiring data has changed. I would like to ask if I can still continue acquiring data, and if so, can I still use Snap for satellite images? I am from Colombia, and during the course, I saw a topic on detecting deforestation or oil spills in water. My intention is to base my thesis on something related to these issues in my country. However, I don't know much about other applications besides Snap, and I'm not sure if there are other tools you use. Also, I'm unsure if I can acquire such images from the same location and create a time series to observe changes. Again, I apologize for so many questions.
